csrc/fusion_segmenter.h

```diff
@@ -132,6 +132,10 @@ class SegmentedGroup {
     return exprs_;
   }
 
+  // Returns a toposorted list of Exprs in this group, with equal cases
+  // respecting the original order.
```

Comment:
I think the name for this is a sort that is considered stable, i.e. equality respects original order

Reply:
Renamed to stablyOrderedExprs